source/fundamentals/auth.txt

```diff
@@ -231,33 +230,90 @@ connect using ``MONGODB-CR``.
-   environment variable.
-4. The default AWS EC2 endpoint. For more information, see `IAM Roles for Tasks
-   <https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html>`__
+1.  Use the AWS SDK (v1 or v2) to authenticate an IAM role.
+#. Specify your AWS credentials in a ``MongoCredential`` instance or connection
+   string.
+#. Specify your AWS credentials in environment variables.
+#. Specify an ECS/EC2 endpoint that hosts your container credentials.
 
+.. important::
 
-The following code snippets show how to specify the authentication mechanism,
-using the following placeholders:
+   The driver uses the preceding order to check for your credentials and uses
+   the first method that it finds.
+
+To learn how to refresh your AWS IAM credentials, see the
+:ref:`<java-mongodb-aws-refresh-credentials>` section.
+
+.. _java-mongodb-aws-sdk:
+
+Authenticate using the AWS SDK
++++++++++++++++++++++++++++++++
+
+You can authenticate with your AWS IAM role using the `AWS SDK for Java <https://docs.aws.amazon.com/sdk-for-java/index.html>`__
+versions 1 or 2 by including it in your project and authenticating with
+the SDK..
+
+If your project includes AWS SDK for Java **v1**, the driver uses the `DefaultAWSCredentialsProviderChain <https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html>`__
+class to specify the credentials.
+
+If your project includes AWS SDK for Java **v2**, the driver uses the `DefaultCredentialsProvider <https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/auth/credentials/DefaultCredentialsProvider.html>`__
+class to specify the credentials.
+
+Authenticate using AWS IAM Credentials
+++++++++++++++++++++++++++++++++++++++
+
+You authenticate with your AWS IAM role by specifying your credentials in
+a connection string, a ``MongoCredential`` instance, or in environment
+variables.
+
+The examples in this section use the following placeholders:
+
+* ``awsKeyId`` - value of your AWS access key ID
+* ``awsSecretKey`` - value of your AWS secret access key
+* ``atlasUri`` - network address of your MongoDB Atlas deployment
+* ``hostname`` - hostname of your MongoDB Atlas deployment
+* ``port`` - port of your MongoDB Atlas deployment
+* ``awsSessionToken`` - value of your AWS session token *(optional)*
 
-* ``awsKeyId`` - value of your ``AWS_ACCESS_KEY_ID``.
-* ``awsSecretKey`` - value of your ``AWS_SECRET_ACCESS_KEY``.
-* ``atlasUri`` - network address of your MongoDB Atlas instance.
-* ``awsSessionToken`` - value of your ``AWS_SESSION_TOKEN``. *(optional)*
+.. _mongodb-aws-connection-string-example:
+
+Connection String Example
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To authenticate using a connection string, assign the ``authMechanism``
+parameter the value ``"MONGODB-AWS"`` and replace the placeholders with your
+credentials in your connection string. The following code example shows how
+you can instantiate a ``MongoClient`` using your connection string:
```

Comment:
Issue: Per [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#term-using), "using" isn't optimal here (the MongoClient could be using your connection string--which, yes, it kind of is, but I don't think that's what we're trying to convey here).

Suggestion:

```suggestion
credentials in your connection string. The following code example shows how
you can use your connection string to instantiate a ``MongoClient``:
```

Reply:
There might be a more accurate and/or concise way to phrase this, but your revised version is unambiguous and understandable, so no changes required here.